nx-X11/programs/Xserver/hw/nxagent/Display.c

```diff
@@ -2169,6 +2169,7 @@ static int nxagentCheckForDefaultDepthCompatibility()
 {
   int dDepth;
 
+  return 1;
```

Comment:
This and the below 'return's addressing deactivations of color depth checks should not be part of this commit, IMHO.

I wonder if Mario's original diff (the first commit of this PR) should appear in the nx-libs history, at all.